app/models/confirmation.rb

```diff
@@ -1,6 +1,8 @@
 class Confirmation
   include ActiveModel::Model
-  attr_accessor :outcome, :message, :vo_number, :no_vo, :no_pvo, :renew_vo, :renew_pvo, :banned_visitors, :unlisted_visitors, :visitor_not_listed, :visitor_banned, :canned_response, :closed_visit
+  attr_accessor :outcome, :message, :vo_number, :no_vo, :no_pvo, :renew_vo,
+    :renew_pvo, :banned_visitors, :unlisted_visitors, :visitor_not_listed,
+    :visitor_banned, :canned_response, :closed_visit
```

Comment:
I wonder if it might be nicer, for future diffing purposes, to just break this line after every `,`. That said, that'll create an incredibly long line. Happy either way, but possibly worth considering.